lib/rules/no-extra-parens.js

```diff
@@ -428,6 +429,38 @@ module.exports = {
             }
         }
 
+        /**
+         * Check the parentheses around the super class of the given class definition.
+         * @param {ASTNode} node The node of class declarations to check.
+         * @returns {void}
+         */
+        function dryClass(node) {
```

Comment:
I think all the other functions in this rule also have the `dry` prefix, although I'm not sure what it means. I agree that `check` would be better.